meta-aos-rpi-domd/recipes-core/udev-rules/udev-rules.bb

```diff
@@ -0,0 +1,13 @@
+SUMMARY = "Udev rules for domd"
```

Comment:
There is recipe udev-rules-rpi.bb in meta-raspberry. I would put it as bbappend to this recipe. Or create new one udev-rules-gpio-rpi.bb under recipes-core/udev folder.